e2e/benchmarks/local-benchmark/index.html

```diff
@@ -312,6 +312,8 @@ <h2>TensorFlow.js Model Benchmark</h2>
     async function warmUpAndRecordTime() {
       if (warmupTimes == 0) {
         await showMsg('Skip warming up');
+        appendRow(timeTable, 'backend', state.backend);
+        appendRow(timeTable, '1st inference', 0);
```

Comment:
It's wired for me that you directly put a zero here. Can't it be the real time to do the first inference? Or Maybe it's better to change the name to `appendRow(timeTable, 'Warmup time', printTime(0));`. And change the `2nd inference` to `Profile time` to make it more readable.

Reply:
Done